docs/user/tutorial/40-people.rst

```diff
@@ -80,11 +84,40 @@ automatically.
 
 We'll create a *People* page, and add an apphook for the Aldryn People application to it.
 
-.. todo:: screenshots of following steps (if they are new steps)
-
 .. todo:: write up steps properly
 
-* create new page called People
-* add Apphook
-* publish page
-* view page with automatic list of people in it
+1. Create new page called People
+
+.. image:: /user/tutorial/images/create_people_page.png
+   :alt: Create new person
+   :width: 600
+   :align: center
+
+2. Select advanced settings
+
+.. image:: /user/tutorial/images/select_advanced_settings.png
+   :alt: Create new person
+   :width: 150
+   :align: center
+
+3. Add Apphook
```

Comment:
as far as i've seen, if there is only one app-hook added to the news, it's set up automatically. if several, additional selectbox appear to choose proper app-hook for current news blog.